### DISCUSSION TITLE
Add your developer account to Xcode.

### DISCUSSION THREAD

**Discussion by ElliotEserinUT** [fastlane/fastlane#21240] [Q&A]
Hi!
I have been making a workflow to build iOS apps with Fastlane in GitHub Actions. I have it working fine with manual signing but wanted to try it with Automatic for various reasons.
However my issue is that When I run an automatic build it says this: "There are no accounts registered with Xcode. Add your developer account to Xcode.". As its a github Action, I dont have access to the xcode instance in order to add an account manually so Im wondering if its possible to do this through code? either in the Action or the Fastfile... I cannot seem to find any documentation on it so Im wondering is it possible to add an accound to Xcode like this? And if so any help or links would be really appreciated!

**Comment by Whearclas34** [comment]
Ensure that you are using a version of Xcode that supports automatic signing. Different versions of Xcode have different capabilities and requirements, make sure you are using a macOS runner (e.g., macos-latest) because Xcode and iOS development are primarily supported on macOS. Consider using Fastlane's match tool for code signing. match is designed to automate the management of certificates and provisioning profiles. It can help you avoid the need to manually add an account to Xcode. Ensure that your GitHub Actions workflow has access to the required Apple Developer Portal credentials. These typically include your App Store Connect username and password. You can store these credentials as secrets in your GitHub repository settings and then use them within your GitHub Actions workflow. Be cautious with handling sensitive data like this and follow best practices for secrets management

**Comment by caldenmic** [comment]
Hi, I wanted to know is there any fastlane support to add accounts to Xcode so that I can automate it easily rather than manually adding the account and entering the OTP (One Time Password) in Xcode. Is there a way we can avoid the OTP entering part by specifying the API key. I hope someone gives more insights.
